### ISSUE TITLE
Add the vertical scroll indicator for better user experience.

### ISSUE THREAD

**Issue by ankit-kmar** [CodeTrophs/Frontend#308]
**Is your feature request related to a problem? Please describe.**
Add the vertical scroll indicator for better user experience.

**Describe the solution you'd like**
A little changes in html, css and js file.

**Describe alternatives you've considered**
Nothing

**Additional context**
Pls assign me this issue with a gssoc21 label and mentioning its level

**Comment by aryasoni98** [comment]
@ankit-kmar Go Head

**Comment by msk4862** [comment]
@ankit-kmar what's the progress?

**Comment by ankit-kmar** [comment]
> @ankit-kmar what's the progress?

yes sir!! actually end sem exams & viva are going on, so will try to do it by tommorow.

**Comment by msk4862** [comment]
@ankit-kmar Any updates?
